examples/config.yaml

```diff
@@ -16,14 +16,17 @@ clusters:
 
     # Client ID - should match value in Dex
     client_id: example-cluster-client-id
-    
+
     # Dex Issuer - Must be resolvable
     issuer:  http://127.0.0.1:5556
 
     # Url to k8s API endpoint - used in WebUI instructions for generating
     # kubeconfig
     k8s_master_uri: https://your-k8s-master.cluster
-    
+
+    # don't use username for context
+    static_context_name: true
```

Comment:
@kirrmann could we default `static_context_name` to `false` to preserve backward compatibility?

Reply:
Sure. I updated it to false in the example config